### ISSUE TITLE
pythonBeans and pythonBeansTests not handling enums containing strings starting with number correctly

### ISSUE THREAD

**Issue by Juha-Ylikoski-Treon** [OkieOth/yacg#46]
Hi

I came across to your awesome looking project and tried to generate python stubs from a json schema. The schema contains field `"Version": "1"` which is enum in my json schema. This is not handled properly creating python files with syntax error.

Example json schema:
```json
{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "title": "Demo",
    "enum": [
        "1"
    ]    
}
```

python output(s)

pythonBeans
```
# Attention, this file is generated. Manual changes get lost with the next
# run of the code generation.
# created by yacg (template: pythonBeans.mako v1.0.0)

from enum import Enum


class Demo(Enum):
    1 = '1'

    @classmethod
    def valueForString(cls, stringValue):
        lowerStringValue = stringValue.lower() if stringValue is not None else None
        if lowerStringValue is None:
            return None
        elif lowerStringValue == '1':
            return Demo.1
        else:
            return None

    @classmethod
    def valueAsString(cls, enumValue):
        if enumValue is None:
            return ''
        elif enumValue == Demo.1:
            return '1'
        else:
            return ''




```

pythonBeansTests
```
# Attention, this file is generated. Manual changes get lost with the next
# run of the code generation.
# created by yacg (template: pythonBeans.mako v1.0.0)

import unittest



class TestTestTitle (unittest.TestCase):
    def testDemo(self):
        self.assertIsNotNone(Demo.1)


if __name__ == '__main__':
    unittest.main()

```

The problem is that in this languages variable name/property etc starting with number is prohibited and raises syntax error. While testing I found that enum containing integers would just crash this program so it is not workaround for me either.

I used command 

```
pipenv run python3 yacg.py --models model.json --singleFileTemplates pythonBeans=tmp/model.py pythonBeansTests=tmp/model.test.py plantUml=tmp/model.plantuml --templateParameters title="Test title";
```

I will probably fix this for myself with some regex pipes or smt but I will leave this issue here for anyone else wondering.

I do not know what would be desired behavior when creating these properties from string containing interger. Maybe some kind of translator which would translate numbers 1,2,3 -> ONE, TWO, THREE

**Comment by OkieOth** [comment]
You are right, I am a lazy in implementing the whole spec :D ... and my knowledge is behind too. I am going to fix this for enums.

**Comment by OkieOth** [comment]
Your descriptions is perfect ... I am going to change name that only consist of numbers

**Comment by OkieOth** [comment]
solved in version 1.3.1
